src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -89,4 +91,34 @@ public static Tag parseTag(String tag) throws ParseException {
         }
         return tagSet;
     }
+
+    /**
+     * Parses a {@code String name} into a {@code Name}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code name} is invalid.
+     */
+    public static ModuleName parseModuleName(String name) throws ParseException {
+        requireNonNull(name);
+        String trimmedName = name.trim();
+        if (!Name.isValidName(trimmedName)) {
+            throw new ParseException(Name.MESSAGE_CONSTRAINTS);
+        }
+        return new ModuleName(trimmedName);
+    }
+
+    /**
+     * Parses a {@code String zoomLink} into a {@code ZoomLink}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code zoomLink} is invalid.
+     */
+    public static ZoomLink parseZoomLink(String zoomLink) throws ParseException {
+        requireNonNull(zoomLink);
+        String trimmedName = zoomLink.trim();
+        if (!Name.isValidName(trimmedName)) {
+            throw new ParseException(Name.MESSAGE_CONSTRAINTS);
```

Comment:
Same issue as above.

Reply:
Thank you for the comment. Have submitted changes for this.